plugins/modules/aci_config_snapshot.py

```diff
@@ -293,6 +293,12 @@ def main():
         # Create a new Snapshot
         aci.post_config()
 
+        # Query for job information and add to results
+        # Change state to query else aci.request() will not execute a GET request but POST
+        aci.params["state"] = "query"
+        aci.request(path="/api/node/mo/uni/backupst/jobs-[uni/fabric/configexp-{0}].json".format(export_policy))
+        aci.result["job_details"] = aci.imdata[0].get("configJobCont", {})
```

Comment:
Maybe we should revert aci.params["state"] after this code is executed. Not sure if changing it changes any of the outputs.

Reply:
Yes indeed you are correct that this impacted the output, also needed to adjust the url. When the state is set to present, the get_existing() method is called which would then fail due to wrong url got set during the request() method.